.github/workflows/main.yml

```diff
@@ -1,11 +1,13 @@
 name: Main
 
-on: pull_request
+on:
+  pull_request:
+  release:
+    types: [published]
 
 jobs:
   build:
     runs-on: ubuntu-latest
```

Comment:
I think we might want to run on Amazon Linux as well, but this can be done later

Reply:
ya maybe in the future, github makes it easy build and test on various platforms (mac and windows) and we could use a self hosted runner for amazon linux.